R/teal_data_module-within.R

```diff
@@ -12,10 +12,10 @@
 #'
 #' @examples
 #' tdm <- teal_data_module(
-#'   ui = function(id) div(id = shiny::NS(id)("div_id")),
+#'   ui = function(id) div(id = NS(id)("div_id")),
```

Comment:
Is this needed? It's essentially an empty div - no value added and actually a little bit of clutter. In order to keep it as simple as possible I would suggest removing. 

(same for eval_code)

```suggestion
#'   ui = function(id) NULL,
```

Reply:
done [0e84058](https://github.com/insightsengineering/teal/pull/1021/commits/0e84058a9bda4b90f98ede76a09d81123720544a)